common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2584,53 +2581,53 @@ func (d *cassandraPersistence) GetTimerIndexTasks(request *GetTimerIndexTasksReq
 	return response, nil
 }
 
-func (d *cassandraPersistence) createTransferTasks(batch *gocql.Batch, transferTasks []Task, domainID, workflowID,
+func (d *cassandraPersistence) createTransferTasks(batch *gocql.Batch, transferTasks []p.Task, domainID, workflowID,
 	runID string) {
 	targetDomainID := domainID
 	for _, task := range transferTasks {
 		var taskList string
 		var scheduleID int64
-		targetWorkflowID := transferTaskTransferTargetWorkflowID
+		targetWorkflowID := p.TransferTaskTransferTargetWorkflowID
 		targetRunID := transferTaskTypeTransferTargetRunID
```

Comment:
if transferTaskTransferTargetWorkflowID is moved out, probably should also move transferTaskTypeTransferTargetRunID to the persistence common package